.travis.yml

```diff
@@ -2,9 +2,12 @@ language: ruby
 sudo: required
 dist: trusty
 rvm:
-  # Stable supported version
+  # Stable supported version (deprecated)
   - 2.4.2
```

Comment:
Do we still need this? Because we raise an exception when initializing the app. So..

Reply:
True...